homeassistant/components/flux_led/light.py

```diff
@@ -139,41 +147,66 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the platform and manage importing from YAML."""
+
+    if config.get("automatic_add", False):
+        await hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": SOURCE_IMPORT}, data={CONF_TYPE: "auto"}
+        )
+
+    else:
+        for import_host, import_item in config.items():
+            if import_host == "platform":
+                continue
+
+            import_name = import_host
+            if import_item:
+                import_name = import_item.get("name", import_host)
+
+            await hass.config_entries.flow.async_init(
+                DOMAIN,
+                context={"source": SOURCE_IMPORT},
+                data={
+                    CONF_TYPE: "manual",
+                    CONF_NAME: import_name,
+                    CONF_HOST: import_host,
+                },
+            )
+
+    return True
```

Comment:
Nothing is checking this return value.
```suggestion
```

Reply:
Fixed - sorry, should have just commited from PR before updating the rest.